src/helpers/data/plugins/i18n/locales/en.json

```diff
@@ -398,12 +398,12 @@
     },
     "success": {
       "tracks": {
-        "header": "All {count} tracks were imported",
+        "header": "All {count} audio tracks have been imported",
```

Comment:
@Borewit I think it should be:
`All {count} tracks have been imported`
In context of application it's just track, not audio track.